content/docs/pipelines/sdk/sdk-overview.md

```diff
@@ -384,7 +384,7 @@ You can also choose to share your pipeline as follows:
 
 {{% alert title="More about the above workflow" color="info" %}}
 For an example of the above workflow, see the
-Jupyter notebook titled [KubeFlow Pipelines basic component build](https://github.com/kubeflow/pipelines/blob/master/samples/core/component_build/component_build.ipynb) on GitHub.
+Jupyter notebook titled [KubeFlow Pipelines basic component build](https://github.com/kubeflow/pipelines/blob/master/samples/core/container_build/container_build.ipynb) on GitHub.
```

Comment:
Suggestion, update the notebook title:

KubeFlow Pipelines basic component build
->
KubeFlow Pipelines container building

Reply:
Thanks @joeliedtke for your review! 

I have modified the file as what you have suggested. Please review/approve. Feel free to let me know if further change is needed.